tests/Functional/Fixtures/web/symfony.php

```diff
@@ -21,4 +21,4 @@
 $request = Request::createFromGlobals();
 $response = $kernel->handle($request);
 $response->send();
-$kernel->terminate($request, $response);
+//$kernel->terminate($request, $response);
```

Comment:
I'm curious if changing the code as suggested in symfony/symfony-standard#658 does the trick.